### PR TITLE
add "repository" link to pyproject

### PR DESCRIPTION
**Pull request by deronnax**
Because link to the Github repository doesn't show up in PyPI and it is annoying.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -30,6 +30,7 @@ dynamic = ["version"]
 Documentation = "https://wheel.readthedocs.io/"
 Changelog = "https://wheel.readthedocs.io/en/stable/news.html"
 "Issue Tracker" = "https://github.com/pypa/wheel/issues"
+Repository = "https://github.com/pypa/wheel/"
```
Comment (henryiii) [inline pyproject.toml:33]: Is this more commonly called `Source`? It usually is in project that call `Homepage` the documentation, like https://pypi.org/project/pip/. Other projects that call documentation `Documentation` call this `Homepage`, like https://pypi.org/project/setuptools/.

Comment (agronholm) [reply pyproject.toml:33]: `Homepage` is annoyingly ambiguous, so let's at least not use that. As for `Repository` vs `Source`, I have no strong opinions one way or the other.

Comment (agronholm) [reply pyproject.toml:33]: Pip uses `Source` so I think we could go with that.

Comment (agronholm) [reply pyproject.toml:33]: Pip uses `Source` so I think we could go with that.

Comment (agronholm) [reply pyproject.toml:33]: Pip uses "Source".
```suggestion
Source = "https://github.com/pypa/wheel/"
```

Comment (agronholm) [reply pyproject.toml:33]: Pip uses "Source".
```suggestion
Source = "https://github.com/pypa/wheel/"
```

File: pyproject.toml [hunk 2]
```diff
@@ -30,6 +30,7 @@ dynamic = ["version"]
 Documentation = "https://wheel.readthedocs.io/"
 Changelog = "https://wheel.readthedocs.io/en/stable/news.html"
 "Issue Tracker" = "https://github.com/pypa/wheel/issues"
+Source = "https://github.com/pypa/wheel/"
```
Comment (agronholm) [inline pyproject.toml:33]:
```suggestion
Source = "https://github.com/pypa/wheel"
```

**Pull request merged**